Documentation/project-docs/glossary.md

```diff
@@ -95,41 +245,90 @@ var odds = source.Where(obj => obj.Id == 1).ToArray();
 
 #### IQueryable&lt;T&gt; and Expressions
 
-One of the big advantages of using LINQ over more common data processing patterns is that the function given to the LINQ function can be converted to an expression and then executed in some other form, like SQL or on another machine across the network. An expression is a in-memory representation of some logic to follow.
+One of the big advantages of using LINQ over more common data processing
+patterns is that the function given to the LINQ function can be converted to an
+expression and then executed in some other form, like SQL or on another machine
+across the network. An expression is a in-memory representation of some logic to
+follow.
 
-For example, in the above sample `source` could actually be a database connection and the function call `Where(obj => obj.Id == 1)` would be converted to a SQL WHERE clause: `WHERE ID = 1`, and then executed on the SQL server.
+For example, in the above sample `source` could actually be a database
+connection and the function call `Where(obj => obj.Id == 1)` would be converted
+to a SQL WHERE clause: `WHERE ID = 1`, and then executed on the SQL server.
 
 #### Parallel LINQ
 
 **Also referred to as**: PLINQ
 
-Also introduced in .NET Framework 3.5 Parallel LINQ. Parallel LINQ has a subset of the methods the LINQ does but may execute the iterations on different threads in any order. Generally to use Parallel LINQ you would just call the `AsParallel()` method on a collection implementing `IEnumerable`. And if at any point you wanted to return to "normal LINQ you can just call `AsSequential()`.
+Also introduced in .NET Framework 3.5 Parallel LINQ. Parallel LINQ has a subset
+of the methods the LINQ does but may execute the iterations on different threads
+in any order. Generally to use Parallel LINQ you would just call the
+`AsParallel()` method on a collection implementing `IEnumerable`. And if at any
+point you wanted to return to "normal LINQ you can just call `AsSequential()`.
 
 ### JSON.NET
 
-Released in June 2006 by [James Newton-King](https://twitter.com/JamesNK), JSON.NET has become the defacto standard for JSON serialization and deserialization in .NET. It is [open source](https://github.com/JamesNK/Newtonsoft.Json) and support almost every platform .NET code can run on (.NET Framework 2.0, 3.0, 3.5, 4.0, and 4.5; Mono; MonoTouch/Xamarin.iOS; MonoDroid/Xamarin.Android; Silverlight 3, 4, and 5; Windows Phone 8, 8, and 8.1; Windows 8 Store; .NET Core).  
+Released in June 2006 by [James Newton-King][JamesNK], JSON.NET has become the
+defacto standard for JSON serialization and deserialization in .NET. It is [open
+source][Newtonsoft.Json] and support almost every platform .NET code can run on
+(.NET Framework 2.0, 3.0, 3.5, 4.0, and 4.5; Mono; MonoTouch/Xamarin.iOS;
+MonoDroid/Xamarin.Android; Silverlight 3, 4, and 5; Windows Phone 8, 8, and 8.1;
+Windows 8 Store; .NET Core).
```

Comment:
In .NET Core 3.0, an all new JSON API will be built into the .NET Core shared framework: System.Text.Json. It is particularly optimized for performance.

Reply:
Should I mention that it's replaced or remove mention at all?